lib/mapper.js

```diff
@@ -142,8 +142,9 @@ function formatRemarketingTag(msg) {
     bundleid: app.namespace,
     rdid: device.advertisingId,
     idtype: getIdType(device),
-    lat: device.adTrackingEnabled ? 1 : 0,
+    lat: device.adTrackingEnabled ? 0 : 1,
     remarketing_only: 1,
+    usage_tracking_enabled: 1,
```

Comment:
Yeah I was about to post this. I think we should just remove that since most customers send events regularly.